lib/vmdb/settings.rb

```diff
@@ -82,6 +88,10 @@ def self.encrypt_passwords!(settings)
       walk_passwords(settings) { |k, v, h| h[k] = MiqPassword.try_encrypt(v) }
     end
 
+    def self.dump_to_log_directory(settings)
+      DUMP_LOG_FILE.write(mask_passwords!(settings.to_hash).to_yaml)
```

Comment:
Gregg,
Thanks for the quick action on this but I do have a question.  I have been
reviewing the current settings file and I see that while the UI log
collection collects all *.txt files, there is no directive to collect any
*.yml file from the vmdb/log directory.  This suggests to me that the log
collection settings will also need to change.  If I am wrong, please advise
otherwise this change creates the updated settings.yml file but nothing
actually collects if for diagnostic purposes.

Tom Hennessy

Thomas Hennessy
Red Hat, Inc
email: thenness@redhat.com
irc: thennessy

On Tue, Aug 23, 2016 at 2:28 PM, Gregg Tanzillo <notifications@github.com>
wrote:

> In lib/vmdb/settings.rb
> <https://github.com/ManageIQ/manageiq/pull/10705#discussion_r75924302>:
>
> > @@ -82,6 +88,10 @@ def self.encrypt_passwords!(settings)
> >        walk_passwords(settings) { |k, v, h| h[k] = MiqPassword.try_encrypt(v) }
> >      end
> >
> > +    def self.dump_to_log_directory(settings)
> > +      DUMP_LOG_FILE.write(mask_passwords!(settings.to_hash).to_yaml)
>
> Nice! 👍🏼
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/10705/files/1510ac05e4c4e40f55d28c48ce8f9e743734664f#r75924302>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD72tvn1HQaW4G6GtLT8arjJpfjf_FkQks5qizvigaJpZM4JrNaH>
> .
>